packages/web-components/src/components/rux-input/rux-input.tsx

```diff
@@ -198,12 +200,27 @@ export class RuxInput implements FormFieldInterface {
     componentWillLoad() {
         this._handleSlotChange()
         this._setTogglePassword()
+
+        const shadowRootNode = this.el.getRootNode() as ShadowRoot
+        this._isDatePicker = shadowRootNode.host ? true : false
```

Comment:
Determines if the input is being used in a datepicker or not. If it is, it adds a class that hides the webkit icon.